platform/platform-scheduler/src/main/java/ddf/platform/scheduler/CommandJob.java

```diff
@@ -53,132 +55,132 @@ protected Subject getSystemSubject() {
   }
 
   @Override
-  public void execute(final JobExecutionContext context) throws JobExecutionException {
-    SECURITY.runAsAdmin(
-        () -> {
-          Subject subject = getSystemSubject();
-
-          if (subject != null) {
-            subject.execute(
-                () -> {
-                  doExecute(context);
-                  return null;
-                });
-          } else {
-            LOGGER.debug("Could not execute command. Could not get subject to run command");
-          }
-
-          return null;
-        });
+  public void execute(final JobExecutionContext context) {
+    try {
+      SECURITY.runAsAdmin(
+          () -> {
+            final String commandString = getCommandString(context);
+            if (StringUtils.isNotBlank(commandString)) {
```

Comment:
❓ If we move `commandString()` outside the `try`, we should consider moving this conditional outside as well and return right away. That would reduce the indentation depth and possibly the cyclomatic complexity of the method.